code/obj/item/plants.dm

```diff
@@ -468,26 +468,59 @@
 				if (istype(H.limbs.r_arm,/obj/item/parts/robot_parts) || istype(H.limbs.r_arm,/obj/item/parts/human_parts/arm/right/synth))
 					..()
 					return
-			if(istype(H))
-				if(H.gloves)
-					..()
-					return
+			if(H.gloves)
+				..()
+				return
 			if(ON_COOLDOWN(src, "prick_hands", 1 SECOND))
 				return
-			boutput(user, "<span class='alert'>You prick yourself on [src]'s thorns trying to pick it up!</span>")
-			random_brute_damage(user, 3)
-			take_bleeding_damage(user,null,3,DAMAGE_STAB)
+			src.prick(user)
 		else
 			..()
 
+	proc/prick(mob/M)
+		boutput(M, "<span class='alert'>You prick yourself on [src]'s thorns trying to pick it up!</span>")
+		random_brute_damage(M, 3)
+		take_bleeding_damage(M,null,3,DAMAGE_STAB)
+
 	attackby(obj/item/W, mob/user)
 		if (issnippingtool(W) && src.thorned)
 			boutput(user, "<span class='notice'>You snip off [src]'s thorns.</span>")
-			src.thorned = 0
+			src.thorned = FALSE
 			src.desc += " Its thorns have been snipped off."
 			return
 		..()
-		return
+
+	attack(mob/living/carbon/human/M, mob/user, def_zone)
+		if (istype(M) && !(M.head?.c_flags & BLOCKCHOKE) && def_zone == "head")
+			M.tri_message(user, "<span class='alert'>[user] holds [src] to [M]'s nose, letting [him_or_her(M)] take in the fragrance.</span>",
+				"<span class='alert'>[user] holds [src] to your nose, letting you take in the fragrance.</span>",
+				"<span class='alert'>You hold [src] to [M]'s nose, letting [him_or_her(M)] take in the fragrance.</span>"
+			)
+			return TRUE
+		..()
+
+/obj/item/plant/flower/rose/poisoned
+	attack(mob/M, mob/user, def_zone)
+		if (!..() || is_incapacitated(M))
+			return
+		src.poison(M, user)
+
+	prick(mob/user)
+		..()
+		src.poison(user)
+
+	proc/poison(mob/M)
+		if (!M.reagents?.has_reagent("capulettium"))
+			if (M.mind?.assigned_role == "Mime")
+				//since this is used for faking your own death, have a little more reagent
+				M.reagents?.add_reagent("capulettium_plus", 20)
+				//mess with medics a little
+				M.bioHolder.AddEffect("dead_scan", timeleft = 40 SECONDS, do_stability = FALSE, magical = TRUE)
+			else
+				M.reagents?.add_reagent("capulettium", 13)
```

Comment:
is 13 some specific time threshold? seems like a weird number

Reply:
Yep, it's the minimum amount that consistently knocks you out before leaving your system (assuming it works consistently with changing life loop speed, which I _think_ it should?) so that it knocks you out for the minimum length of time (still quite long)